docs/data/material/getting-started/templates/landing-page/getLPTheme.tsx

```diff
@@ -0,0 +1,592 @@
+      light: red[50],
+      main: red[500],
+      dark: red[700],
+      ...(mode === 'dark' && { light: '#D32F2F', main: '#D32F2F', dark: '#B22A2A' }),
+    },
+    success: {
+      light: green[300],
+      main: green[400],
+      dark: green[800],
+      ...(mode === 'dark' && {
+        light: green[400],
+        main: green[500],
+        dark: green[700],
+      }),
+    },
+    grey: {
+      50: gray[50],
+      100: gray[100],
+      200: gray[200],
+      300: gray[300],
+      400: gray[400],
+      500: gray[500],
+      600: gray[600],
+      700: gray[700],
+      800: gray[800],
+      900: gray[900],
+    },
+    divider: mode === 'dark' ? alpha(gray[600], 0.3) : alpha(gray[300], 0.5),
+    background: {
+      default: '#fff',
+      paper: gray[50],
+      ...(mode === 'dark' && { default: gray[900], paper: gray[800] }),
+    },
+    text: {
+      primary: gray[800],
+      secondary: gray[600],
+      ...(mode === 'dark' && { primary: '#fff', secondary: gray[400] }),
+    },
+    action: {
+      selected: `${alpha(brand[200], 0.2)}`,
+      ...(mode === 'dark' && {
+        selected: alpha(brand[800], 0.2),
+      }),
+    },
+  },
+  typography: {
+    fontFamily: ['"Inter", "sans-serif"'].join(','),
+    h1: {
+      fontWeight: 600,
+      lineHeight: 78 / 70,
+      fontSize: 60,
+      letterSpacing: -0.2,
+    },
+    h2: {
+      fontSize: 48,
+      fontWeight: 600,
+      lineHeight: 1.2,
+    },
+    h3: {
+      fontSize: 42,
+      lineHeight: 1.2,
+    },
+    h4: {
+      fontSize: 36,
+      fontWeight: 500,
+      lineHeight: 1.5,
+    },
+    h5: {
+      fontSize: 20,
+      fontWeight: 600,
+    },
+    h6: {
+      fontSize: 18,
+    },
+    subtitle1: {
+      fontSize: 18,
+    },
+    subtitle2: {
+      fontSize: 16,
+    },
+    body1: {
+      fontWeight: 400,
+      fontSize: 15,
+    },
+    body2: {
+      fontWeight: 400,
+      fontSize: 14,
+    },
+    caption: {
+      fontWeight: 400,
+      fontSize: 12,
+    },
+  },
+});
+
+export default function getLPTheme(mode: PaletteMode): ThemeOptions {
+  return {
+    ...getDesignTokens(mode),
+    components: {
+      MuiAccordion: {
+        defaultProps: {
+          elevation: 0,
+          disableGutters: true,
+        },
+        styleOverrides: {
+          root: ({ theme }) => ({
+            padding: 8,
+            overflow: 'clip',
+            backgroundColor: '#fff',
+            border: '1px solid',
+            borderColor: gray[100],
+            ':before': {
+              backgroundColor: 'transparent',
+            },
+            '&:first-of-type': {
+              borderTopLeftRadius: 10,
+              borderTopRightRadius: 10,
+            },
+            '&:last-of-type': {
+              borderBottomLeftRadius: 10,
+              borderBottomRightRadius: 10,
+            },
+            ...theme.applyDarkStyles({
+              backgroundColor: gray[900],
+              borderColor: gray[800],
+            }),
```

Comment:
```suggestion
            ...theme.palette.mode === 'dark' && {
              backgroundColor: gray[900],
              borderColor: gray[800],
            },
```

`theme.applyDarkStyles` is an internal function for our doc to help transitioning to CSS variables, it hasn't been exposed to the public yet.

For template related code, I suggest using the check above and apply the styles directly.

In the near future, we plan to expose `theme.applyStyles()` with codemod, so these templates will be great examples.